### PR TITLE
Sort variables after renaming them. (PT-185139443)

### PR DESCRIPTION
**Pull request by lublagg**
This PR fixes an issue in which if a user tried to rename a wedge to the same name as another wedge but if they were not adjacent to one another, different wedges with the same name to appear on the Spinner. This PR uses the already-existing sortVariablesForSpinner function to fix this issue.

### CODE REVIEW

File: TP-Sampler/src/view.js [hunk 1]
```diff
@@ -952,13 +953,15 @@ View.prototype = {
         variables[editingVariable] = "";
       } else {
         var v = variables[editingVariable];
+        console.log({v});
```
Comment (eireland) [inline TP-Sampler/src/view.js:956]: Remove console.log

File: TP-Sampler/src/view.js [hunk 2]
```diff
@@ -952,13 +953,15 @@ View.prototype = {
...
         editingVariable = [];
         for (var j = 0, jj = variables.length; j < jj; j++) {
           if (variables[j] === v) {
             // variables[j] = " ";
             editingVariable.push(j);
           }
         }
+        console.log({editingVariable});
```
Comment (eireland) [inline TP-Sampler/src/view.js:964]: remove console.log

File: TP-Sampler/src/view.js [hunk 3]
```diff
@@ -952,13 +953,15 @@ View.prototype = {
         variables[editingVariable] = "";
       } else {
         var v = variables[editingVariable];
+        console.log({v});
         editingVariable = [];
         for (var j = 0, jj = variables.length; j < jj; j++) {
           if (variables[j] === v) {
             // variables[j] = " ";
```
Comment (eireland) [inline TP-Sampler/src/view.js:960]: remove commented out code

**Review by eireland — APPROVED** [review]
Remove a couple of console.logs and a commented out line of code.
Otherwise looks good!

File: TP-Sampler/src/view.js (hunk 3 above)
Comment (lublagg) [reply TP-Sampler/src/view.js]: thanks for catching these!

**Pull request merged**